tests/e2e/e2e-tests/block_volume_attach.go

```diff
@@ -143,6 +138,19 @@ func fileExists(filename string) (bool, error) {
 	return true, nil
 }
 
+func getAttchStatus() (string, error) {
+	attachStatus := "attaching"
+	err := errors.New("Timed out in PV creation")
+	for start := time.Now(); time.Since(start) < (10 * time.Minute); {
+		attachStatus = pv.ObjectMeta.Annotations["ibm.io/attachstatus"]
+		if attachStatus == "attached" || attachStatus == "failed" {
+			return attachStatus, nil
+			break
```

Comment:
Remove this break as its never called since you return from previous line

Reply:
ok